### DISCUSSION TITLE
How to define a new LVAR ?

### DISCUSSION THREAD

**Discussion by JLD95** [maartentamboer/X-Touch-Mini-FS2020#103] [Q&A] [answered]
Hi,

I would to have a button for XPDR mode but L:A32NX_SWITCH_TCAS_POSITION seems not working and editor display "not available"
How to add this variable ?  Should I use a mobiflight define ?

![Presse-papiers-1](https://user-images.githubusercontent.com/5894146/164960758-5624dafe-5b66-493a-9931-8e7dc9aa7f5c.jpg)

**Comment by Koseng** [comment, marked as answer]
Hi,
writing to a variable is done via a simconnect event. Events need to defined and registered before they can be used.

The mobiflight events are defined in the file "events.txt" in your mobiflight addon in the community folder. You can add something there and then access it via xtouchmini. After adding something to events.txt, the simulator needs to be restarted. In the xtouch script you use the identifier before the "#" in the events file with the prefix "MobiFlight.".

If you have something really useful which is missing you can add it to the mobiflight variable and events collection on https://hubhop.mobiflight.com/. From the hubhop the events.txt is generated. And future versions will contain your new definition.

**Reply by JLD95** [reply]
Great this URL, thanks
